src/main/java/net/phptravels/gui/pages/LoginPage.java

```diff
@@ -27,14 +27,36 @@ public LoginPage(WebDriver driver){
       setUiLoadedMarker(emailField);
   }
 
-  public DashboardPage loginFillForm(String email,String password) {
-	  emailField.click();
-	  emailField.type(email);
-	  passwordField.click();
-	  passwordField.type(password);
-	  loginButton.click();
-   
-    return new DashboardPage(driver);
-  }
+    public ExtendedWebElement getEmailField() {
+        return emailField;
+    }
+
+    public void setEmailField(String emailField) {
+        this.emailField.type(emailField);
+    }
+
+    public ExtendedWebElement getPasswordField() {
```

Comment:
do not return ExtendedWebElement from page objects
all actions with element should be inside page object class